### ISSUE TITLE
`Development`: PapaParse new function of duplicated headers conflicts with the array.extension.ts last method

### ISSUE THREAD

**Issue by MaximilianJG** [ls1intum/Artemis#8271]
### Describe the bug

PR #8215 exposes an issue with duplicated headers, which is out of scope of that PR. 
PapaParse 5.4.1 introduces renaming of duplicated headers, which breaks because of it conflicts with the array.extension.ts 'last' method'

### To Reproduce

1. Go to detailed-grading-system.component.spec.ts 
2. Create a duplicated field (or rather header) in the invalid CSV. (e.g. `4.0,10,10,TRUE`)
3. Run the test, and notice that when logging the results.data the output contains code from the array.extension.ts file) 

### Expected behavior

Normally, PapaParse would just rename the duplicated headers. 
E.g. for 4.0,10,10,True → 4.0, 10_1, 10_2, True

### Which version of Artemis are you seeing the problem on?

6.9.2

### What browsers are you seeing the problem on?

Other (specify in "Additional context")

### Additional context

Client Test

### Relevant log output

```shell
console.log
    [
      { '4.0': '        if (!this.length) {' },
      { '4.0': '            return undefined;' },
      { '4.0': '        }' },
      { '4.0': '        return this[this.length - 1];' },
      { '10': 'function first() {', '4.0': '    }' },
      { '4.0': '        if (!this.length) {' },
      { '4.0': '            return undefined;' },
      { '4.0': '        }' },
      { '4.0': '        return this[0];' },
      { '10': 'function shuffle() {', '4.0': '    }' },
      { '4.0': '        if (this.length > 1) {' },
      { '4.0': '            for (let i = this.length - 1; i > 0; i--) {' },
      {
        '4.0': '                const randomIndex = Math.floor(Math.random() * (i + 1));'
      },
      { '4.0': '                const randomElement = this[randomIndex];' },
      { '4.0': '                this[randomIndex] = this[i];' },
      { '4.0': '                this[i] = randomElement;' },
      { '4.0': '            }' },
      { '4.0': '        }' },
      { '4.0': '    }' }
    ]
```

**Comment by krusche** [comment]
Any idea why this happens?

**Comment by MaximilianJG** [comment]
Unfortunately not, deleting the content of the array.extension file seems to fix the problem; so I am assuming it has to do with that...

**Comment by krusche** [comment]
our array extension was actually problematic. I fixed it in e0fe2db075a24499ff68c2405ea219c02fddc144 and updated the dependency
